__tests__/app.test.js

```diff
@@ -84,3 +84,47 @@ describe("/api/users", () => {
     });
   });
 });
+describe("/api/articles/:article_id", () => {
+  describe("PATCH", () => {
+    test("200: Updates the votes property with specified amount and responds with the updated article ", () => {
+      const patchRequest = { inc_votes: 1 };
+      return request(app)
+        .patch("/api/articles/1")
+        .send(patchRequest)
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.article.votes).toBe(101);
+        });
+    });
+    test("400: Responds with bad request if article_id is invalid ", () => {
```

Comment:
This test description is for an invalid article_id but that isnt the invalid part of this request